src/Core/Grid/Definition/Factory/OrderGridDefinitionFactory.php

```diff
@@ -144,6 +145,18 @@ protected function getColumns()
                     'field' => 'id_order',
                 ])
             )
+            ->add(
+                (new PreviewColumn('preview'))
+                    ->setOptions([
+                        'text_type' => 'static',
+                        'static_text' => '<',
+                        'identifier_field' => 'id_order',
+                        'preview_data_route' => 'admin_orders_get_preview_data',
+                        'preview_record_params' => [
```

Comment:
How about `preview_route_params`?

Reply:
wanted to point out that values are taken from records. In case somebody needs to "hardcode" the parameter directly in gridDefinitionFactory?